rollup.types.config.js

```diff
@@ -0,0 +1,39 @@
+import dts from 'rollup-plugin-dts';
+import styles from 'rollup-plugin-styles';
```

Comment:
I'm a bit confused by this config. I would think it's just about generating types, but then there's this `rollup-plugin-styles` module in it as well. Could you elaborate on this?